client/src/helpers/APICalls/contest.ts

```diff
@@ -4,43 +4,44 @@ import { Contest } from '../../interface/Contest';
 import axios from 'axios';
 
 export const getAllContests = async (): Promise<AuthApiData> => {
-    const fetchData: FetchOptions = {
-        method: 'GET',
-        headers: { 'Content-Type': 'application/json' },
-        credentials: 'include'
-    }
+  const fetchData: FetchOptions = {
+    method: 'GET',
+    headers: { 'Content-Type': 'application/json' },
+    credentials: 'include',
+  };
 
-    return await fetch('/contest', fetchData)
-        .then(data => data.json())
-        .catch(err => ({ error: { message: 'Can not connect to server' } }))
-}
+  return await fetch('/contest', fetchData)
+    .then((data) => data.json())
+    .catch((err) => ({ error: { message: 'Can not connect to server' } }));
```

Comment:
I just randomly notice a typo here, 'Cannot' 😄

Reply:
This is fixed.